### ISSUE TITLE
Support for `sails.config.sockets.onConnect` will be removed in an upcoming release.

### ISSUE THREAD

**Issue by Vadorequest** [balderdashy/sails#2999]
I get this warning in the console when I lift my app.

```
debug: Deprecation warning: Support for `sails.config.sockets.onConnect` will be removed in an upcoming release. See the v0.11 migration guide for more information and alternate options.
debug: Deprecation warning: `sails.config.sockets.onDisconnect` is now `sails.config.sockets.afterDisconnect`
Setting it for you this time, but note that the new `afterDisconnect` now receives an additional final argument (a callback).
More info: http://sailsjs.org/#/documentation/reference/sails.config/sails.config.sockets.html
```

(https://github.com/balderdashy/sails/blob/master/0.11-migration-guide.md)

I actually don't understand why using `onConnect` is bad, from your explanations.

In the other hand, `afterDisconnect` seems to replace `onDisconnect`.

**Comment by shbm1505** [comment]
I have also the same issue....i get this error with command sails lift.

**Comment by sailsbot** [comment]
Thanks for posting, @Vadorequest.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
